src/Compilers/Server/VBCSCompilerTests/ClientConnectionHandlerTests.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO.Pipes;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CommandLine;
+using Moq;
+using Roslyn.Test.Utilities;
+using Xunit;
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+using System.IO;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using static Microsoft.CodeAnalysis.CommandLine.BuildResponse;
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.CompilerServer.UnitTests
+{
+    public class ClientConnectionHandlerTests
+    {
+        [Fact]
+        public async Task ThrowDuringBuild()
+        {
+            var compilerServerHost = new TestableCompilerServerHost(delegate { throw new Exception(); });
```

Comment:
>delegate { throw new Exception(); } [](start = 68, length = 35)

Do you _have_ to use the `delegate` syntax?

Reply:
Until we support multiple discards in lambdas ... yes :smile: